requirements.txt

```diff
@@ -1,5 +1,4 @@
 # Django/Framework Packages
-django>=1.8,<1.9
```

Comment:
@efischer19 I have put the requirement back in. The problem I had before was that test_requirement.txt installed everything from requirement.txt as well, and that caused a duplicate definition with the tox files definition.

I fixed up test_requirement.txt and modified the Makefile to install both requirements files explicitly.